starters/lit-mobx-vaadin/package.json

```diff
@@ -13,6 +13,7 @@
     "prettier": "eslint --ext .ts,.html . --fix --ignore-path .gitignore && prettier \"**/*.ts\" --write --ignore-path .gitignore",
     "storybook": "tsc && npm run analyze -- --exclude dist && concurrently -k -r \"tsc --watch --preserveWatchOutput\" \"wds -c .storybook/server.mjs\"",
     "storybook:build": "tsc && npm run analyze -- --exclude dist && build-storybook",
+    "test:unit": "tsc && wtr --coverage",
     "test": "playwright test",
```

Comment:
I'm wondering if we should make the unit tests the default instead of the integration tests since as far as I know that's how all the other starts work.

Reply:
updated